rt_entt_codegen/core/print_sys_exec.cc

```diff
@@ -171,35 +172,43 @@ static auto print_sys_exec_ctx_remove(
 			type_name,
 			">(this, ecsact_id_cast<ecsact_component_like_id>(",
 			type_name,
-			"::id));\n"
+			"::id), *view);\n"
 		);
 		return;
 	}
-	block(
-		ctx,
-		"static const auto remove_fns = "
-		"std::unordered_map<ecsact_component_like_id, "
-		"decltype(&ecsact_system_execution_context_remove)>",
-		[&] {
-			for(int i = 0; i < remove_comps.size(); ++i) {
-				const auto comp_id = remove_comps[i];
-				auto       type_name = cpp_identifier(decl_full_name(comp_id));
-				ctx.write(
-					"{ecsact_id_cast<ecsact_component_like_id>(",
-					type_name,
-					"::id), &wrapper::dynamic::context_remove<",
-					type_name,
-					"> },"
-				);
-			}
+	ctx.write(std::format(
+		"using remove_fn_t = void (*)(ecsact_system_execution_context*, "
+		"ecsact_component_like_id, {}_t&);\n",
+		view_name
+	));
+
+	ctx.write("static const auto remove_fns = []()\n");
+
+	block(ctx, "", [&] {
+		ctx.write(
+			"auto result = std::unordered_map<ecsact_component_like_id, "
+			"get_fn_t>{};\n"
+		);
```

Comment:
this doesn't seem right. It's using `get_fn_t`, but we're in the remove fn. And the `remove_fn_t` is using something the `view_name` with `_t`? I don't think I see the generated alias for that in here.